simdev/module/git/clone_stage.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import os
+import textwrap
+
+from git import GitCommandError
+from pydriller import Repository
+from tqdm import tqdm
+
+from simdev.util.pipeline import Pipeline, Stage, PipelineException
```

Comment:
import order here and everywhere - you can use [sort imports from pycharm](https://www.jetbrains.com/pycharm/guide/tips/optimize-imports/)

* sort `from` & `import` together
* case-insensetive
* join imports

Reply:
That's the way my instance of PyCharm sorts & arranges all the imports